resources/migrations/001_update_migrations.yaml

```diff
@@ -8191,6 +8191,50 @@ databaseChangeLog:
               column:
                 name: dashboard_id
 
+  # last_used_at is used to determine if a report card is stale and should be cleaned up
+  # if the card is imported with serialization, we don't want to automatically clean up
+  # newly imported cards, and we don't want to have to make it nullable.
+  - changeSet:
+      id: v50.2024-07-09T20:04:01
+      author: calherries
+      comment: Populate default value for report_card.last_used_at
+      changes:
+        - sql:
+            sql: >-
+              UPDATE report_card
+              SET last_used_at = current_timestamp
```

Comment:
Because `last_used_at` can be `NULL` from cards imported with serialization, and we don't want to automatically clean up these cards just because they were imported, we have to set `last_used_at` to the `current_timestamp`.